tests/experimental_combination/integration_testing/main.py

```diff
@@ -45,23 +44,21 @@
 
 
 def get_wzdx():
-    return [json.loads(open('./tests/data/wzdx.json').read())]
+    return [json.loads(open("./tests/data/wzdx.json").read())]
 
-    query = _datastore_client.query(
-        namespace='rtdh_cache_latest', kind='RTDHCache')
-    query.add_filter('label', '=', 'CacheWZDxPlannedEventsStandard')
-    query.add_filter('condition_1', '=', True)
+    query = _datastore_client.query(namespace="rtdh_cache_latest", kind="RTDHCache")
```

Comment:
Is any of this needed? We return right above this so this code isn't reachable.

Reply:
removed this whole directory, it is from the rtdh repos, not useful here